### PR TITLE
added ExpectNoAttributeValueSpec section

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: test/specs/ExpectNoAttributeValueSpec.ts [hunk 1]
```diff
@@ -3,27 +3,39 @@ import { BrowserUtils } from '../../src/commons/BrowserUtils';
...
+  export const EMPTY_DIV: string = '//*[@id="noAttDiv"]';
 }
 /**
  * wdio-allure-ts ExpectNoAttributeValueSpec action test
  */
 describeCommon('expectNoAttributeValue', () => {
   it("Doesn't contains value", () => {
     expect(() =>
-      BrowserUtils.expectNoAttributeValue(PageLocator.FORM, 'method', 'pos')
+      BrowserUtils.expectNoAttributeValue(
```
Comment (FelixZilber) [inline test/specs/ExpectNoAttributeValueSpec.ts]: consider adding test on not existing element

Comment (Ad1B3nAr0ya) [reply test/specs/ExpectNoAttributeValueSpec.ts]: done

Comment (Ad1B3nAr0ya) [reply test/specs/ExpectNoAttributeValueSpec.ts]: not working properly removing

Comment (FelixZilber) [reply test/specs/ExpectNoAttributeValueSpec.ts]: what's not working properly?

Comment (Ad1B3nAr0ya) [reply test/specs/ExpectNoAttributeValueSpec.ts]: Chai expect doesn't catch the error

**Pull request merged**